src/client/img/buttonMinusSvg.tsx

```diff
@@ -0,0 +1,10 @@
+import { FunctionComponent } from 'react';
```

Comment:
So this might seem weird, why not just use the SVG file as an `<img>`?  Because I am I need to use these in different color variations. When embedding SVG directly in the DOM, I can trivially change the color using CSS. Not possible with an `img` tag. Let me know if I am missing a better way.

Reply:
I mean, I can have multiple copies of the same SVG in different colors if that is preferable. Doesn't work if we ever need to change the color on hover though